docker-compose.yml

```diff
@@ -3,8 +3,6 @@ version: '2'
 services:
   zookeeper:
     image: confluentinc/cp-zookeeper:5.0.0
-    hostname: zookeeper
-    container_name: zookeeper
```

Comment:
@rmoff : what's the purpose to remove hostname and container_name?  They are useful in identify the containers, e.g. in output of `docker-compose ps`

Reply:
The container will inherit the service name, which is why specifying these two in addition is redundant

Reply:
Running the docker-compose with your suggested changes results in this:

```bash
/tmp: docker-compose ps
        Name                     Command            State                           Ports                         
------------------------------------------------------------------------------------------------------------------
connect                 /etc/confluent/docker/run   Up      0.0.0.0:8083->8083/tcp, 9092/tcp                      
tmp_kafka_1             /etc/confluent/docker/run   Up      0.0.0.0:9092->9092/tcp                                
tmp_schema-registry_1   /etc/confluent/docker/run   Up      0.0.0.0:8081->8081/tcp                                
tmp_zookeeper_1         /etc/confluent/docker/run   Up      2181/tcp, 2888/tcp, 0.0.0.0:32181->32181/tcp, 3888/tcp
```

Notice the names inherit the directory name, which is what we do not want.  We want it to be called just `zookeeper`, etc

Reply:
Docker compose uses the directory name as a prefix for the label of the container. 
But, you can still reference it with just the service name: 

```
$ docker-compose restart zookeeper
Restarting kafka-connect-datagen_zookeeper_1 ... done
```

It's up to you if you want to keep these, it was just for brevity that I suggest removing them as they're not actually needed.

Reply:
@rmoff I'm usually in favor of brevity as well, but in this case the brevity hurts the naming of the container which should be very clear.  IMO, let's leave `hostname` and `container_name` in the docker-compose file